src/inputcard.ts

```diff
@@ -62,6 +76,25 @@ document.addEventListener('DOMContentLoaded', () => {
     }
   }
 
+  // Custom validator for CVV
+  function checkCvvSize(): void {
+    const size = (creditcardformSecurecode && creditcardformSecurecode.getAttribute('data-validator-size')) || '3';
```

Comment:
I can't understand how the right size of the security code is computed as it's known to depend on country and circuit

Reply:
Every time the credit number field is modified, a correct size of cc is set with a data-attributed on the secure code field, cause the credit-card number (circuit) decided how long could be the secure code (CID in Amex, CVV for the other one)